src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -233,10 +238,202 @@
+    <Compile Include="mono\System.Drawing\ColorTranslator.cs" />
+    <Compile Include="mono\System.Drawing\ComIStreamMarshaler.cs" />
+    <Compile Include="mono\System.Drawing\ComIStreamWrapper.cs" />
+    <Compile Include="mono\System.Drawing\ContentAlignment.cs" />
+    <Compile Include="mono\System.Drawing\CopyPixelOperation.cs" />
+    <Compile Include="mono\System.Drawing\Font.cs" />
+    <Compile Include="mono\System.Drawing\FontFamily.cs" />
+    <Compile Include="mono\System.Drawing\Graphics.cs" />
+    <Compile Include="mono\System.Drawing\GraphicsUnit.cs" />
+    <Compile Include="mono\System.Drawing\gdipEnums.cs" />
+    <Compile Include="mono\System.Drawing\gdipFunctions.cs" />
+    <Compile Include="System\Drawing\GdiplusNative.Unix.cs" />
+    <Compile Include="mono\System.Drawing\gdipStructs.cs" />
+    <Compile Include="mono\System.Drawing\Icon.cs" />
+    <Compile Include="mono\System.Drawing\ImageAnimator.cs" />
+    <Compile Include="mono\System.Drawing\Image.cs" />
+    <Compile Include="mono\System.Drawing\KnownColors.cs" />
+    <Compile Include="mono\System.Drawing\Pen.cs" />
+    <Compile Include="mono\System.Drawing\Pens.cs" />
+    <Compile Include="mono\System.Drawing\Region.cs" />
+    <Compile Include="mono\System.Drawing\SRDescriptionAttribute.cs" />
+    <Compile Include="mono\System.Drawing\StringFormat.cs" />
+    <Compile Include="mono\System.Drawing\SystemColors.cs" />
+    <Compile Include="mono\System.Drawing\SystemFonts.cs" />
+    <Compile Include="mono\System.Drawing\SystemIcons.cs" />
+    <Compile Include="mono\System.Drawing\SystemPens.cs" />
+    <Compile Include="mono\System.Drawing\ToolboxBitmapAttribute.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\GraphicsContainer.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\GraphicsPath.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\LinearGradientBrush.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\Matrix.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\PathGradientBrush.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\BitmapData.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\ImageFormat.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\Metafile.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\MetafileHeader.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\MetaHeader.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintingServices.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintingServicesWin32.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintingServicesUnix.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PageSettings.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PreviewPrintController.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintController.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintDocument.cs">
+      <SubType>Component</SubType>
+    </Compile>
```

Comment:
For some reason, Visual Studio keeps adding this into the project file. I tried to remove it a couple of times. It has something to do with the base type, I think.